src/verifier/step3/Step3Data.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: Apache-2.0
+// Do not change manually. This contract has been auto-generated by src/verifier/step3/step3-data-contractgen.py
+pragma solidity ^0.8.0;
+
+library Step3Data {
+/* This function returns the following results in order:
+* pp_digest, NIFS instance, r_U comm_W, r_U comm_E, r_U X, r_U u
+* l_u comm_W, l_u X
+*/
+function returnData() public pure returns (uint256, uint256[] memory, uint256, uint256, uint256[] memory, uint256, uint256, uint256[] memory) {
```

Comment:
You probably had noticed, but: indentation.

Reply:
https://github.com/lurk-lab/solidity-verifier/pull/15